### PR TITLE
Add "Adobe ColdFusion 2018" to trycf engines and set it as default

### LINKED ISSUE #902: Run Code missing engine
When clicking `Run Code` for an example, the pop-over of **TryCF** does not have "Adobe ColdFusion 2018" as a selectable engine even though it is available on the TryCF website.

### PR DESCRIPTION
**Pull request by shaedrich**
Resolve #902

### CODE REVIEW

File: assets/code-editor.js [hunk 1]
```diff
@@ -106,12 +106,13 @@ angular.module('code.editor', [])
...
-	'				     <option value="acf2016">Adobe ColdFusion 2016</option>'+
-	'				     <option value="acf11">Adobe ColdFusion 11</option>'+
-	'		             <option value="acf">Adobe ColdFusion 10</option>'+
+	'		             	<option value="lucee5">Lucee 5.LATEST</option>'+
+	'		             	<option value="lucee">Lucee 4.5.LATEST</option>'+
+	'		             	<option value="railo">Railo 4.2</option>'+
+	'				<option value="acf2018">Adobe ColdFusion 2018</option>'
+	'				<option value="acf2016">Adobe ColdFusion 2016</option>'+
+	'				<option value="acf11">Adobe ColdFusion 11</option>'+
+	'		             	<option value="acf">Adobe ColdFusion 10</option>'+
```
Comment (pfreitag) [inline assets/code-editor.js]: You also need to add a URL for cf2018 look around line 213

Comment (pfreitag) [reply assets/code-editor.js]: Looks like that URL should be https://acf13-sbx.trycf.com/getremote.cfm

Comment (shaedrich) [reply assets/code-editor.js]: This url is not available. Neither with acf13 nor acf18 in subdomain.

**Comment by pfreitag** [comment]
Hmm - I just tested this locally and for some reason all the ACF engines except CF2018 are missing from the drop down list now. Not sure why, your changes looked good  to me.

**Comment by shaedrich** [comment]
Well, that's very strange…

**Pull request merged**